tools/quickstart/quickstart/netstats.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import click
+import requests
+
+from quickstart.constants import NETSTATS_ENV_FILE_PATH, NETSTATS_SERVER_BASE_URL
+from quickstart.utils import is_netstats_prepared
+
+ENV_FILE_TEMPLATE = """\
+WS_USER={username}
+WS_PASSWORD={password}
+INSTANCE_NAME={instance_name}
+# HIDE_VALIDATOR_STATUS=false
```

Comment:
This was the origin part by Ralf. I would rather not touch it, because this is form master and must have some reason for that. Unresolve it, if you have strong feelings about it.